new/PRS13277Abbelo.xml

```diff
@@ -56,10 +56,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <nationality type="Egypt"/>
                     <birth/>
                     <death/>
-                    <floruit>According to a narrative in <ref type="mss" corresp="EMML1840">he was a monastic figure
+                    <floruit>According to a narrative in <ref type="mss" corresp="EMML1840">EMML 1840</ref> he was a monastic figure
                     contemporary to <persName ref="PRS2209Athanasi">Athanasius of Alexandria</persName>. In the same text he is referred to as
                   "resembling angels" (<foreign xml:lang="gez">ዘይመስል፡ መላእክት፡</foreign>) which might suggest an identification
-                with Apollo mentioned in the Coptic Synaxary on the 25th of Bābah.</ref></floruit>
+                with Apollo mentioned in the Coptic Synaxary on the 25th of Bābah.</floruit>
```

Comment:
for saints/martyrs it is always helpful to mention diesNatalis according to the date of commemoration in the Synaxary or Gadla Sama'tat